tests/tpch/conftest.py

```diff
@@ -136,20 +288,54 @@ def spark(spark_setup, benchmark_time):
     spark_setup.catalog.clearCache()
 
 
+@pytest.fixture
+def fs(local):
+    if local:
+        return None
+    else:
+        import boto3
+        from pyarrow.fs import S3FileSystem
+
+        session = boto3.session.Session()
+        credentials = session.get_credentials()
+
+        fs = S3FileSystem(
+            secret_key=credentials.secret_key,
+            access_key=credentials.access_key,
+            region="us-east-2",
+            session_token=credentials.token,
+        )
+        return fs
+
+
 #################################################
 # Single machine fixtures for Polars and DuckDB #
 #################################################
 
 
-machine = {  # TODO: figure out where to place this
-    "vm_type": "m6i.8xlarge",
-}
-
-_uuid = uuid.uuid4().hex[:8]
+@pytest.fixture(scope="session")
+def machine_spec(scale):
+    if scale == 10:
+        return {
+            "vm_type": "m6i.8xlarge",
+        }
+    elif scale <= 100:
```

Comment:
```suggestion
    elif scale == 100:
```

Not that it actually matters. ;)

Reply:
Done.  Thanks.